inc/blocks/ab-test/components/variant-panel.js

```diff
@@ -60,6 +84,15 @@ const VariantPanel = ( { title, variant, variants, onMouseDown } ) => {
 					} );
 				} }
 			/>
+			{ !! postId && (
+				<Preview
+					href={ `/?p=${ postId }&set_test=${ blockId }&i=${ index }` }
```

Comment:
This link format isn't quite right, this is the regex currently used to check the value: `(utm_campaign|set_test)=test_${ this.testIdWithPost }:(\d+)`, where `testIdWithPost` would be `xb_<xb post id>`.

Reply:
I understand what you're doing here now, so it's achieving the same thing more or less but avoiding the need to fetch the XB post in the back end just to get the ID. I think it's ok to stick with this.